BoxBrowseSDK/BoxBrowseSDK/Helpers/BOXBrowseSDKFileTypeHelper.h

```diff
@@ -0,0 +1,26 @@
+//
+//  BOXBrowseSDKFileTypeHelper.h
+//  BoxBrowseSDK
+//
+//  Created by Clement Rousselle on 1/3/17.
+//  Copyright © 2017 BOX. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface BOXBrowseSDKFileTypeHelper : NSObject
+
++ (NSSet <NSString *> *)audioFileExtensions;
++ (NSSet <NSString *> *)imageFileExtensions;
++ (NSSet <NSString *> *)vectorImageFileExtensions;
++ (NSSet <NSString *> *)videoFileExtensions;
++ (NSSet <NSString *> *)docFileExtensions;
++ (NSSet <NSString *> *)codeFileExtensions;
++ (NSSet <NSString *> *)textFileExtensions;
++ (NSSet <NSString *> *)compressedFileExtensions;
++ (NSSet <NSString *> *)presentationFileExtensions;
++ (NSSet <NSString *> *)sheetFileExtensions;
++ (NSSet <NSString *> *)dbFileExtensions;
++ (NSSet <NSString *> *)iconFileExtensions;
+
```

Comment:
What about box-dicom or box-notes? Would we treat them differently or handle them the same way?